apps/versions/tests.py

```diff
@@ -833,6 +840,22 @@ def test_success(self):
         assert self.addon.current_version
         self.assert_served_by_mirror(self.client.get(self.latest_url))
 
+    def test_beta(self):
+        r = self.client.get(self.latest_beta_url)
+        eq_(r.status_code, 302)
+        beta_file_url = reverse('downloads.file', args=[self.beta_file.id])
+        url = beta_file_url + '/' + self.beta_file.filename
+        assert r['Location'].endswith(url), r['Location']
```

Comment:
You don't need the `, r['Location']`, because we're using [pytest](http://pytest.org/latest/assert.html?highlight=introspection) and its clever "assert introspection", which gives you all the needed details if a test fails, for free!